<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="maBase" uuid="d9d7de86-18ec-4989-b3ac-b31e4b674971">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\Steeven\Documents\Cours_DEVOPS\pythonProject\TpInterventionTechnicien\DataAccess\maBase.db</jdbc-url>
      <libraries>
        <library>
          <url>file://$APPLICATION_CONFIG_DIR$/jdbc-drivers/Xerial SQLiteJDBC/3.34.0/sqlite-jdbc-3.34.0.jar</url>
        </library>
      </libraries>
    </data-source>
    <data-source source="LOCAL" name="maBase driver - identifier.sqlite" uuid="7fd8b15c-2738-4e64-8203-ee0bd5e530a1">
      <driver-ref>a985d6ea-0161-4088-98f4-309864b913d1</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:identifier.sqlite</jdbc-url>
    </data-source>
  </component>
</project>